packages/firebase_mlkit_language/test/firebase_mlkit_language_test.dart

```diff
@@ -0,0 +1,7 @@
+// import 'package:flutter/services.dart';
```

Comment:
Please include unit tests and integration tests if possible; see [Firebase ML Vision](https://github.com/flutter/plugins/tree/master/packages/firebase_ml_vision/example/test_driver) for an example.